lib/index.js

```diff
@@ -74,17 +74,10 @@ export default {
 
         // Call ERB to "de-templatize" the code
         return helpers.exec(this.erbPath, erbArgs,
-          { stdin: text, cwd: fileDir }
+          { stdin: text.replace(/<%=/g, '<%'), cwd: fileDir }
```

Comment:
I just checked and there is another scope called "HTML (Rails)" but in all the ERB files that I have in my project it wasn't used. It's always "HTML (Ruby - ERB)" which I suppose is `text.html.erb`. Oh, and the previous check would always pass `text.html.erb` scopes to the replacement function. I personally use this: `if (scopes.indexOf('text.html.erb') > -1)` which for me is easier to read then using the not equals.

The only problem is that this invalid regular ERB code would not raise an error:

```erb
<%= something or_other do %>
<% end %>
```

But, including the `if (scopes.indexOf('text.html.erb') !== -1) {` line wouldn't help anyway since it seems that the scope is always `text.html.erb`, so it always gets passed. For that to work only on rails ERB it would be required that Atom identified rails ERB files as such.

So, there's nothing to do here. If some user of regular ERB complains about the missing error we'll revisit this, but it requires changes outside of `linter-erb`.